training/README.md

```diff
@@ -0,0 +1,60 @@
+Training in Digital Publishing
+===========================
+
+:warning: **This section is very much a work in progress. Links may not resolve, and content linked to may be unfinished** 
+
+When joining Digital Publishing, there can be a lot to learn! We want to give you the benefit of all our previous learning, so we've tried to gather it in one place.
+
+We have collected resources that give you an idea of [all the modules](#modules) some of which you'll be undertaking over your first few weeks. These may also contain links to resources the team have found useful over the years to get more insight on the topics.
+
+If you come across a resource that really helped you understand some technology we use, or how we work, please do add a link to it in the relevant document or start a new doc if the topic is not yet covered.
+
+Modules
+----------------------------
+#### Team culture & process
+* [Culture and Inclusion]()
+* [What is Digital Publishing?](culture-and-process/DIGITAL_PUBLISHING.md) 
+* [Developer Processes]()
```

Comment:
Where does 70/20/10 time come in, continuous development, pull from the right etc

Reply:
70/20/10 probably culture, but called out as its own thing, possibly link to a document about it